governance/roles.md

```diff
@@ -0,0 +1,96 @@
+
+## Chair
+
+The (normally three) Co-Chairs of the TAG App Delivery drive the CNCF community efforts of the TAGs domain.
+Chairs are experienced leaders within the CNCF community with rooted expertise in the TAG area. Chairs serve a two-year term but may renew their term by submitting a follow-up application.
+Chair applications go through an [election process](https://github.com/cncf/toc/blob/master/tags/cncf-tags.md#elections) with a final CNCF TOC vote. The election process is defined [here](./chair-election-process.md)(WIP).
+
+Co-Chairs are expected to:
+* Manage the operations and governance of the group.
+* Organize, host, plan and facilitate TAG meetings and events.
+* Reporting to the CNCF TOC on the status of TAG work.
+* Encouraging community members to start projects and working groups.
+* Resolve technical difficulties and decisions related to multiple sub-projects.
+* Remind the community of the scope of the TAG and point out guardrails in discussions, working groups, and projects.
+* Serve as TAG leadership representative in CNCF project discussions and at community events.
+* Evolve the TAG to reflect ongoing changes in the industry.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and promote diversity in TAG work.
+* Enforce and promote good communication in TAG efforts in accordance with the [CNCF CoC](https://www.cncf.io/conduct/)
+
+Permissions:
+* Chairs have admin access to the TAG GitHub repository.
+
+## Technical Lead
+
+A technical lead (TL) expands the bandwidth of the leadership team in terms of covering technical areas, growing the community, organizing events or doing TAG communications.
+
+Proposals must have Chair as an active sponsor and the agreement of the majority of the current leadership team (Chairs, Tech Leads & Liaisons). TLs are experienced contributors within the CNCF community with rooted expertise in the TAG area. 
+
+Technical leads go through an [proposal process](./tech-lead-proposal-process.md) with a final CNCF TOC vote.
+
+**A technical lead is expected to**:
+* Establish and guide projects and working groups.
+* Resolve cross-sub-project technical difficulties and decisions.
+* Propose agenda items for meetings to ensure that open issues are discussed with the group when needed.
+* Serving as TAG leadership representative in CNCF project discussions and on community events.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and encourage diversity in the TAG efforts.
+* Enforce and encourage good communication in the TAG efforts following the [CNCF CoC](https://www.cncf.io/conduct/).
+* Support the TAG Chairs.
+
+Permissions:
+* Tech Leads have maintain access to the TAG GitHub repository.
+
+## Working Group Leads
+
+A working group chairs (usually 3) are experienced contributors to the TAG that applies to lead along one or two others a working group.
```

Comment:
Perhaps - "Up to 3 chairs are appointed to lead each working group."

Should we define what we mean by "experienced"?